lib/mayaUsd/render/vp2RenderDelegate/proxyRenderDelegate.cpp

```diff
@@ -319,6 +320,12 @@ void ProxyRenderDelegate::_InitRenderDelegate(MSubSceneContainer& container) {
             }
         }
     }
+    else
```

Comment:
GitHub doesn't allow me to place a comment at line 255, but this is where I would expect this check to happen.

Otherwise, I would expect _isInitialized() at line 258 return true and we never get till here to check exclude prims version.